### ISSUE TITLE
a11y - Search results thumbnails need alt text

### ISSUE THREAD

**Issue by mberkowski** [UMNLibraries/cdm-blacklightify#104]
### Descriptive summary
Thumbnails need alt text
* Search result thumbs
* Homepage Spotlight collection thumbs

**Comment by mberkowski** [comment]
Also More Like This from item view.

**Comment by mberkowski** [comment]
<img width="637" height="621" alt="Image" src="https://github.com/user-attachments/assets/c25ea9d3-fa6e-4c36-9ab0-54f77dd7c65d" />

<img width="853" height="349" alt="Image" src="https://github.com/user-attachments/assets/40fdf214-6cb2-4b00-babf-eb90b9081643" />
